bot/utils/time.py

```diff
@@ -111,3 +111,33 @@ def parse_rfc1123(stamp: str) -> datetime.datetime:
 def format_infraction(timestamp: str) -> str:
     """Format an infraction timestamp to a more readable ISO 8601 format."""
     return dateutil.parser.isoparse(timestamp).strftime(INFRACTION_FORMAT)
+
+
+def get_duration_from_expiry(
+    expiry: str = None,
+    date_from: datetime.datetime = None,
+    max_units: int = 2
+) -> Optional[str]:
+    """
+    Returns a human-readable version of the  the duration between datetime.utcnow() and an expiry.
```

Comment:
```suggestion
    Returns a human-readable version of the duration between datetime.utcnow() and an expiry.
```

Reply:
Thank you, I will include this in new commits to avoid having too many small commits.